### ISSUE TITLE
Can't minimize Deepin apps after selecting "Window effect"

### ISSUE THREAD

**Issue by KimHono** [linuxdeepin/dde-control-center#346]
Hi !
I noticed this curious issue in the Control Center:
Go in `Personalization`, select `General`, then activate `Window Effect`.

And now, open any Deepin app (file manager, terminal, PDF reader...): it's possible to minimize this window only 1 time.
Further tries by clicking on "minimize" button will have no effect on this app window !

**Comment by N1coc4colA** [comment]
Have you installed latest updates? I haven't such problem on my Deepin installation... I can open close, and minimize any time I want to. Have you check about KWin?

**Comment by KimHono** [comment]
Yes, I'm using deepin-kwin 5.2.0.13-2, but finally I'm always working with desactivated Windows effects. In this situation, there is no problem :roll_eyes:
